File: Login/app/src/main/java/com/vishcustom/login/MainActivity.java
package com.vishcustom.login;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;
import androidx.appcompat.app.AppCompatActivity;

public class  MainActivity extends AppCompatActivity {

    private EditText etUsername, etPassword;
    private Button btnSignIn;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        etUsername = findViewById(R.id.etUsername);
        etPassword = findViewById(R.id.etPassword);
        btnSignIn = findViewById(R.id.btnSignIn);

        btnSignIn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String username = etUsername.getText().toString();
                String password = etPassword.getText().toString();

                // Perform authentication logic here (e.g., check credentials)
                if (username.equals("vishakha") && password.equals("12345")) {
                    // Successful authentication
                    Toast.makeText(MainActivity.this, "Sign-in successful", Toast.LENGTH_SHORT).show();
                } else {
                    // Failed authentication
                    Toast.makeText(MainActivity.this, "Sign-in failed", Toast.LENGTH_SHORT).show();
                }
            }
        });
    }
}

File: mark01/app/src/main/java/com/vishcustom/button/MainActivity.kt
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

import android.widget.Button
import android.widget.EditText
import android.widget.Toast
import com.vishcustom.mark01.R

class MainActivity : AppCompatActivity() {
    private lateinit var editUserId: EditText
    private lateinit var editPassword: EditText
    private lateinit var btnLogin: Button

    private val PRESCRIBED_USER_ID = "your_user_id"
    private val PRESCRIBED_PASSWORD = "your_password"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        editUserId = findViewById(R.id.editUserId)
        editPassword = findViewById(R.id.editPassword)
        btnLogin = findViewById(R.id.btnLogin)

        btnLogin.setOnClickListener {
            val enteredUserId = editUserId.text.toString()
            val enteredPassword = editPassword.text.toString()

            if (enteredUserId == PRESCRIBED_USER_ID && enteredPassword == PRESCRIBED_PASSWORD) {
                showToast("Login Successful")
            } else {
                showToast("Login Failed. Please check your credentials.")
            }
        }
    }

    
    private fun showToast(message: String) {
        Toast.makeText(this, message, Toast.LENGTH_SHORT).show()
    }
}
